README.md

```diff
@@ -75,13 +75,36 @@ This is one of three components required for Project Lightspeed. Project Lightsp
 
 ## Getting Started
 
+## Setup
+
+### Docker
+
+1. Install [git](https://git-scm.com/downloads)
+1. Build the image from the master branch with:
+
+    ```sh
+    docker build -t grvydev/lightspeed-react https://github.com/GRVYDEV/Lightspeed-react.git
+    ```
+
+1. Run it with
+
+    ```sh
+    docker run -it --rm \
+      -p 8000:80/tcp -p 8080:8080/tcp \
+      -e WEBSOCKET_HOST=localhost -e WEBSOCKET_PORT=8080 grvydev/lightspeed-react
+    ```
+
+1. You can now access it at [localhost:8000](http://localhost:8000) and the websocket port is published on port `8080`.
```

Comment:
Note: using port `8000` to show that you can use another port for people not familiar with Docker

Reply:
See other comment (I might be wrong). This might be misleading that `lightspeed-react` has something to do with webrtc on port `8080` by saying "websocket port is published"

Reply:
Oh yes good point, I'll fix up the readme.

Reply:
I changed the WEBSOCKET_HOST to `lightspeed-websocket-host` assuming it would run also in a container in the same docker network. If a better name comes up, please comment here 😉